### ISSUE TITLE
Screen reader reads an unlabeled button next to the 'pick a file' button on the submission screen.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3238]
**Describe the bug**

Screen reader reads an unlabelled button next to the 'pick a file' button on the submission screen.

**To Reproduce**
"1. Go to the Sample challenge page 
2. Ensure that the screen reader is switched on. 
3. Activate the 'submit' link after the registration process is completed. 
4. Press the key combination 'ctrl+insert+b' to bring up a list of buttons on the page.
5. Notice that there is an 'unlabelled3' button listed next to the 'pick a file' button."

**Expected behavior**

All buttons should have an appropriate label to indicate its purpose to the user.

**Actual behavior**
Screen reader announces an unlabelled3 button next to the pick a file button.

**Additional context**

Type: Submission

Page: https://www.topcoder.com/challenges/30095347

Status: Unclear, this element is now being read as 'select file to upload tab, not sure of its functionality.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099910 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099910 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099910 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@veshu looks like the issue is fixed already. Unable to view any unlabelled button. Please review.
